src/rgw/rgw_admin.cc

```diff
@@ -2289,7 +2289,7 @@ static void parse_tier_config_param(const string& s, map<string, string, ltstr_n
   }
 }
 
-static int check_pool_support_omap(rgw_pool pool) 
+static int check_pool_support_omap(const rgw_pool &pool) 
```

Comment:
nit: in rgw, we generally put the & next to the type, as in `const rgw_pool& pool`

Reply:
Thanks for pointing it out! There didn't seem to a convention for this across files, and I tried to stick with what I saw in each file. I missed this one.